### ISSUE TITLE
quote value in `ConsoleRender` like `LogfmtRenderer` for special chars

### ISSUE THREAD

**Issue by trim21** [hynek/structlog#647]
`logger.info("hello", a="1 b=2")` will be logged as `hello1 a=1 b=2` and may cause some problem, maybe we should quote value when it contain any space ` `

for example:

```
class _ConsoleRender(structlog.dev.ConsoleRenderer):

    def _repr(self, val: Any) -> str:
        v = str(val)
        if " " in v or '\n' in v:
            return repr(v)

        return v
```

or just return `repr(val)`

**Comment by hynek** [comment]
Would you mind opening a PR?

**Comment by trim21** [comment]
> Would you mind opening a PR?

I can open a PR, but I'm not sure which solution you prefer. simple repr or only quote spaces?

**Comment by trim21** [comment]
I prefer simple `repr(val)` for all values.

**Comment by trim21** [comment]
OK, did't notice there is already `repr_native_str` option exists.

**Comment by hynek** [comment]
fixed by #649
